_posts/2020-05-21-#19010.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: pr
+date: 2020-05-21
+title: "BIP 157: Add support for getcfheaders"
+pr: 19010
+authors: [jnewbery]
```

Comment:
add jimpo as co-author in the authors array?

Reply:
Yes of course. Good catch.